include/swift/AST/Decl.h

```diff
@@ -5008,6 +5008,9 @@ class FuncDecl final : public AbstractFunctionDecl,
   }
 
   bool isGetter() const { return getAccessorKind() == AccessorKind::IsGetter; }
+  bool isComputedGetter() const {
```

Comment:
You're quick! Let me try the check you suggested, sorry about that. What would be the correct name for it?